tests/test_typing.cpp

```diff
@@ -4,6 +4,29 @@
 namespace nb = nanobind;
 using namespace nb::literals;
 
+class NestedClass {};
+
+namespace nanobind {
+namespace detail {
+template <>
+struct type_caster<NestedClass> {
+    NB_TYPE_CASTER(NestedClass, const_name("py_stub_test.AClass.NestedClass"));
+
+    bool from_python(handle /*src*/, uint8_t /*flags*/, cleanup_list*) noexcept {
+        return true;
+    }
+
+    static handle from_cpp(const NestedClass&, rv_policy, cleanup_list*) noexcept {
+        nanobind::object py_class =
+            nanobind::module_::import_("py_stub_test").attr("AClass").attr("NestedClass");
+        nanobind::object py_proto = py_class();
+        py_proto.inc_ref();
```

Comment:
These two lines can be shortened to: `return py_class().release()`

Reply:
Done